cognite_toolkit/_cdf_tk/prototypes/commands/dump_timeseries.py

```diff
@@ -0,0 +1,306 @@
+from __future__ import annotations
+
+import shutil
+from collections.abc import Iterator
+from pathlib import Path
+from typing import Any, Literal, cast
+
+import pandas as pd
+import questionary
+import yaml
+from cognite.client import CogniteClient
+from cognite.client.data_classes import (
+    DataSetWrite,
+    DataSetWriteList,
+    TimeSeriesFilter,
+    TimeSeriesList,
+)
+from cognite.client.exceptions import CogniteAPIError
+from rich.progress import Progress, TaskID
+
+from cognite_toolkit._cdf_tk.commands._base import ToolkitCommand
+from cognite_toolkit._cdf_tk.exceptions import (
+    ToolkitFileExistsError,
+    ToolkitIsADirectoryError,
+    ToolkitMissingResourceError,
+    ToolkitValueError,
+)
+from cognite_toolkit._cdf_tk.loaders import DataSetsLoader
+from cognite_toolkit._cdf_tk.prototypes.resource_loaders import TimeSeriesLoader
+from cognite_toolkit._cdf_tk.utils import CDFToolConfig
+
+#
+# TODO: what is the naming convention for TimeSeries, timeseries, time_series, singular or plural?
```

Comment:
In the Python-SDK we use TimeSeries, and time_series. Unfortunately, there is no nice way to distinguish singular and plural, but can use Hungarian notation (as you have done) with `time_series_list` and `time_series_lists`.

Reply:
thx for pointing `time_series_lists` out, helped to replace my odd `one_timee_series` name :)